libsupport/src/ArrowVisitor.cpp

```diff
@@ -102,8 +225,8 @@ struct ToArrayVisitor : public katana::ArrowVisitor {
 katana::Result<void>
 katana::AppendToBuilder(
     const arrow::Scalar& scalar, arrow::ArrayBuilder* builder) {
-  katana::AppendScalarToBuilder visitor(builder);
```

Comment:
I've never had to do it, but I'm assuming there are parts of the code where it is necessary/convenient to use the same visitor over multiple builders?